fs/base.py

```diff
@@ -568,7 +568,7 @@ def getbytes(self, path):
             contents = read_file.read()
         return contents
 
-    def getfile(self, path, file, chunk_size=None, **options):
+    def download(self, path, file, chunk_size=None, **options):
```

Comment:
`getfile` has always felt awkward as it doesn't *return* a file, like `gettext` returns text.

From a network filesystem this makes total sense, it may not be quite so obvious for other filesystems, but I think it actually fits.